### ISSUE TITLE
ARAX returning multihop for treats queries

### ISSUE THREAD

**Issue by edeutsch** [RTXteam/RTX#2187]
URGENT question from the sponsor:

For a treats query:
![image](https://github.com/RTXteam/RTX/assets/12707718/284665f9-1073-4506-b025-1692254a9876)

https://arax.ci.transltr.io/?r=174832

ARAX is returning things like this:
![image](https://github.com/RTXteam/RTX/assets/12707718/8513b52d-6955-408d-9b06-3153d1e04067)

It looks like it it is multiple forms of MS.

Is that legal or a problem?

**Comment by isbluis** [comment]
FWIW, and if it helps: this same query has been posted to our system over 80 times in the past ~6 weeks, and the results seem to be similar going back to Sept.13: https://arax.transltr.io/index.html?r=168073

**Comment by edeutsch** [comment]
I think the result should just be original MONDO:0005301 and the other forms of MS can be listed in support graphs.
So I think this is not right.

But I am not certain.

**Comment by edeutsch** [comment]
> FWIW, and if it helps: this same query has been posted to our system over 80 times in the past ~6 weeks, and the results seem to be similar going back to Sept.13: https://arax.transltr.io/index.html?r=168073

Do you mean that it is correct before Sep 13 or no information before Sep 13?

**Comment by amykglen** [comment]
yeah, those are subclasses of MS. the 'subclass_of' edges indicating that appear in support graphs.

was a standard way of handling subclass relationships in results ever decided on/implemented? (last I knew before the summer it had been tabled..)

**Comment by amykglen** [comment]
if we remap the 'treats' edges that go to a subclass of MS to point to MS itself, users won't know which treats edge pointed to which subclass of MS, even if we still include subclass_of edges in support graphs. right?

I think that's why we chose to do it the way we're doing it (and is at the heart of why this got tabled by Architecture back in spring?) but maybe there's now a consensus that it's ok to lose that information (?)

**Comment by edeutsch** [comment]
yes, thank you, this is hazily coming back to me now. I think you're right that we (Translator) declined to put this in subgraphs

**Comment by amykglen** [comment]
got it. yeah, it definitely looks weird without the subclass_of edges appearing in the Result itself, but we're not allowed to put those in the Result itself since they don't fulfill any qedge in the original QG.

I suppose that's a matter of display though...

**Comment by amykglen** [comment]
alternatively - I suppose maybe the sponsor would prefer that we create _separate_ results for MS and each of its subclasses? but that seems less helpful to me...

**Comment by saramsey** [comment]
OK, where did we land on this issue, in terms of TRAPI validity?  Is ARAX returning valid TRAPI for the originally referenced query?

**Comment by edeutsch** [comment]
I think I pushed back a little on the private Slack discussion that brought this up, saying that we have not yet agreed Translator-wide to put subclass inferencing in support graphs, so our understanding is that this is the way to do it. No one has corrected me so far. So we might be okay.

I'll go ahead and close. Reopen if necessary.
this is a TRAPI 1.5 discussion topic, so maybe it will come up then.
